$NetBSD: patch-ac,v 1.1 2007/03/17 14:52:51 rillig Exp $

gcc4 complains:
regs.h:28: error: array type has incomplete element type
regs.h:48: error: array type has incomplete element type

--- regs.h.orig	2003-01-20 14:53:08.000000000 +0100
+++ regs.h	2007-03-17 15:50:38.000000000 +0100
@@ -3,6 +3,8 @@
 #ifndef __REGS_H
 #define __REGS_H
 
+#include "z80-asm.h"	/* for struct seznam_type */
+
 #define R_EMPTY 0
 #define R_A 1
 #define R_B 2
